### DISCUSSION TITLE
With the update  ==3.14.0 ->  ==3.15.0, ModelAdmin must define "search_fields"

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [encode/django-rest-framework#9316] [Potential Issue] [answered]
Recently upgraded to v3.15.0 and unable to run the server `./manage.py runserver` or `./manage.py migrate`. I get the error:

```
django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:

ERRORS:
<class 'rest_framework.authtoken.admin.TokenAdmin'>: (admin.E040) ModelAdmin must define "search_fields", because it's referenced by TokenAdmin.autocomplete_fields.

System check identified 1 issue (0 silenced).
```

I tried to create a custom admin class in my `admin.py` file but it still gave me this error. 

The only thing worked for me was to open the `.venv/lib/python3.12/site-packages/rest_framework/authtoken/admin.py` and inside the  `TokenAdmin`, commenting or deleting the `autocomplete_fields`.

```
class TokenAdmin(admin.ModelAdmin):
    ...
    search_fields = ('user__username',)
    search_help_text = _('Username')
    ordering = ('-created',)
    actions = None  # Actions not compatible with mapped IDs.
    # autocomplete_fields = ("user",)

    def get_changelist(self, request, **kwargs):
        return TokenChangeList
   ...
```

How to address this issue properly? I can't do this manual commenting out the package class variable. 

Any help is appreciated.

**Comment by browniebroke** [comment, marked as answer]
Duplicated of https://github.com/encode/django-rest-framework/issues/9300

**Reply by i-huzaifa-arshad** [reply]
Well, it was not kind of duplicated of #9300 as my error code is different. But thanks, somehow the issue is temporarily fixed.
